api/seeds.py

```diff
@@ -25,6 +41,35 @@ def seeds():
             print(f"Failed to update services: {exc}")
 
 
+@app.cli.command("dev-db")
```

Comment:
Curious what your thoughts would be on:

1) Changing the name to `local-seeds` or `seeds-local`, something derivative of the `seeds` command? Or a flag to run `seeds --local`? 
2) If we didn't reuse `seeds --local` and still have a separate command, could we reuse the service seeding logic?

Reply:
Brilliant. Glad you suggested that. I went down a rabbit hole and figured it out.